### PR TITLE
#297: Modify to show empty string, if Iterable or array is empty

### PR DESCRIPTION
**Pull request by btshimizukza**
I've fixed bug. see [here](https://github.com/terasolunaorg/terasoluna-gfw/issues/297#issuecomment-121821531).

Please review #297 .

### CODE REVIEW

**Comment by making** [comment]
please rebase

**Comment by kazuki43zoo** [comment]
I've rebased.

File: terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/el/ObjectToMapConverter.java [hunk 1]
```diff
@@ -170,10 +170,15 @@ public ObjectToMapConverter(FormattingConversionService conversionService) {
      * Convert the given Iterable to the flatten map
      * @param prefix prefix of the key
      * @param value iterable instance to convert
-     * @return converted map. all keys are prefixed with the given key
+     * @return converted map. all keys are prefixed with the given key.
+     * If given Iterable is empty, add the pair of the given name(prefix) and empty string into map of return value.
      */
     private Map<String, String> convert(String prefix, Iterable value) {
         Map<String, String> map = new LinkedHashMap<String, String>();
+        if (!value.iterator().hasNext()) {
```
Comment (making) [inline terasoluna-gfw-web/src/main/java/org/terasoluna/gfw/web/el/ObjectToMapConverter.java]:
``` java
Iterator iterator = value.iterator();

if (!iterator.hasNext()) {
  // ...
}
for (int i = 0; iterator.hasNext(); i++) {
  Object o = iterator.next();
  // ...
}
```

is more effective.

**Comment by kazuki43zoo** [comment]
Thanks for good advice.
I've modified iteration logic. Please review agin.

**Comment by making** [comment]
please backport

**Pull request merged**